src/ngx_http_lua_ssl_ocsp.c

```diff
@@ -490,7 +490,6 @@ ngx_http_lua_ffi_ssl_set_ocsp_status_resp(ngx_http_request_t *r,
 
     dd("set ocsp resp: resp_len=%d", (int) resp_len);
     (void) SSL_set_tlsext_status_ocsp_resp(ssl_conn, p, resp_len);
-    ssl_conn->tlsext_status_expected = 1;
```

Comment:
So if this wasn't clear, the correct way to set this variable is having the status callback function return SSL_TLSEXT_ERR_OK, which seems it always does anyway. It looks like setting this variable here doesn't do anything useful in any version of OpenSSL and obviously doesn't work anymore in 1.1.0. So I believe this part of the patch is correct.